draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -315,6 +314,25 @@ Any claims used that are not understood MUST be ignored.
 Additional validation rules MAY apply, but their use is out of the scope of this
 specification.
 
+## Public Key for Issuer-signed JWTs Verification {#public-key-for-issuer-signed-jwts-verification}
+
+A recipient of the SD-JWT VC MUST apply the following rules to verify the
+Issuer-signed JWT corresponds to the Issuer:
+- JWT Issuer Metadata: If the `iss` value contains a HTTPS URI, the recipient MUST
+obtain the public key using JWT Issuer Metadata as defined in (#jwt-issuer-metadata).
+- DID Document Resolution: If the `iss` value contains a DID, the recipient MUST retrieve
+the public key from the DID Document resolved from the DID in the `iss` value.
+If the `kid` JWT header parameter is present, the `kid` MUST be a relative or absolute
+DID URL of the DID in the `iss` value, identifying the public key.
+- X.509 Certificates: The recipient MUST obtain the public key from the leaf X.509 certificate
+defined by the `x5c`, `x5c`, or `x5t` JWT header parameters of the Issuer-signed JWT and validate the X.509
+certificate chain in the following cases:
+    - If the `iss` value contains a DNS name encoded as a URI using the DNS URI scheme [@RFC4501]. The DNS name MUST matche a `dNSName` Subject Alternative Name (SAN) [@RFC5280] entry of the leaf certificate.
+    - If the `iss` value contains a URN using the URN URI scheme [@RFC2141]. The URN MUST match a `unifiedResourceName` SAN entry of the leaf certificate.
+
+Additional rules for verifying the public key belongs to the Issuer MAY be used but are
+out of scope of this specification.
```

Comment:
```suggestion
Separate specifications or ecosystem regulations MAY define rules substituting or complementing the rules defined above, but such rules are
out of scope of this specification. See (#ecosystem-verification-rules) for security considerations.
```

Reply:
I would disagree with this. I don't think that ecosystems should introduce breaking changes. They MAY add rules on top of what was defined. Additionally, it is always a verifier decision whether or not a public key verification method is acceptable. This is a policy decision. Therefore I would keep the language. Does this make sense?

Reply:
The current text does not mean that verifiers have to blindly accept SD-JWT VCs with specific verification methods if it is against their policy.

Reply:
I'm fine with removing the "substituting or" part. I don't think we should keep the "MAY be used" part in the original text as it opens to door quite a bit too wide.

Reply:
Sounds good. I'll adjust the text.